app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+class MoviesController < ApplicationController
+  def index
+    @movies = Movie.all.map do |movie|
+      movie_params(movie)
+    end
+    render json: @movies
+  end
+
+  def show
+    movie = Movie.find(params[:id])
+    @transformed_movie = movie_params(movie)
+    render json: @transformed_movie
+  end
+
+  def new
```

Comment:
You do not need this action if you create only an API application :)

Reply:
Actually instead of `new` it should be `create` action. `new` action is redundant